### ISSUE TITLE
Fetch InstrumentationScope from ReadableSpan

### ISSUE THREAD

**Issue by bobstrecansky** [open-telemetry/opentelemetry-php#705]
The spec compliance matrix defines [Fetch InstrumentationScope from ReadableSpan](https://github.com/open-telemetry/opentelemetry-specification/blob/9abbdd39d0b35f635f833f072013431da419894e/specification/common/mapping-to-non-otlp.md#instrumentationscope)

We need to implement this.

**Comment by Grunet** [comment]
@bobstrecansky it looks like @brettmc updated this for the Jaeger and Zipkin cases at some point per [this search](https://github.com/open-telemetry/opentelemetry-php/search?q=otel.scope.name).

Is the concern that this needs to be added to other non-OTLP exporter cases in the repo?

Or maybe that the instrumentation library piece needs to be added back for backwards compatibility? (for what exactly I'm not really clear. Maybe like if Jaeger and Zipkin have been configured to look for the instrumentation library piece perhaps already, it's not gonna work with just sending the instrumentation scope piece? Also I find it somewhat hard to follow the conversation on [the original PR](https://github.com/open-telemetry/opentelemetry-specification/pull/2276) that updated the spec)

Or maybe other things?

**Comment by bobstrecansky** [comment]
This issue is mostly a placeholder to ensure that all of that spec compliant matrix is completed.  Looks like you've done a bunch of sleuthing; I believe this is done for us.  That deprecated bit shouldn't matter for us, maybe?  We should probably instrument those as well for consistency's sake.

**Comment by brettmc** [comment]
It looks like we can fetch the instrumentation scope from a span, so as Bob said we just needed to verify that we have implemented per spec, and you could submit a PR against the compliance matrix to complete this issue.

**Comment by Grunet** [comment]
Spun up https://github.com/open-telemetry/opentelemetry-specification/pull/2622 on the assumption that the breaking changes the other libraries have to worry about are about direct consumers relying on the old instrumentation library piece, not backends (from looking through a few more PRs of other language libraries linked to that main spec PR, erlang being the only one I've seen so far that did the straight swap without deprecating).
